pkgs/applications/science/molecular-dynamics/lammps/default.nix

```diff
@@ -1,54 +1,63 @@
 { stdenv, fetchFromGitHub
 , libpng, gzip, fftw, openblas
+, cmake, pkgconfig, python2
 , mpi ? null
 }:
-let packages = [
-     "asphere" "body" "class2" "colloid" "compress" "coreshell"
-     "dipole" "granular" "kspace" "manybody" "mc" "misc" "molecule"
-     "opt" "peri" "qeq" "replica" "rigid" "shock" "snap" "srd" "user-reaxc"
-    ];
-    lammps_includes = "-DLAMMPS_EXCEPTIONS -DLAMMPS_GZIP -DLAMMPS_MEMALIGN=64";
-    withMPI = (mpi != null);
+let
+  withMPI = (mpi != null);
+
+  packages = [
+    "asphere" "body" "class2" "colloid" "compress" "coreshell" "dipole"
+    "granular" "kspace" "manybody" "mc" "meam" "misc" "molecule" "opt"
+    "peri" "poems" "qeq" "replica" "rigid" "shock" "snap" "srd" "user-atc"
+    "user-fep" "user-manifold" "user-misc" "user-molfile" "user-reaxc"
+    "kokkos" "python"
+  ]
+  ++ stdenv.lib.optionals withMPI [ "mpiio" ];
+
+  lammps_includes = [
+    "-DLAMMPS_EXCEPTIONS=yes"
+    "-DLAMMPS_GZIP=yes"
+    "-DLAMMPS_MEMALIGN=64"
+    "-DBUILD_LIB=yes"
+  ];
+
 in
 stdenv.mkDerivation rec {
   # LAMMPS has weird versioning converted to ISO 8601 format
-  version = "stable_22Aug2018";
+  version = "stable_7Aug2019";
```

Comment:
I saw upstream uses these weird version schemes,. (sigh). I tend to think perhaps we should use a `2019-08-07` instead? I wonder if `nix` would even detect _this_ update as a new version..